### PR TITLE
Update release verification scripts and incorporate into source control

### LINKED ISSUE #9: Update release verification for new naming conventions in install scripts
The server tar.gz no longer has the version in the file name.  Release verification should be updated to use the new naming conventions.  Additionally, release verification should be integrated into the source control repository.

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,20 +32,90 @@ cd quipucords-installer
 ./install.sh
```
Comment (myersCody) [inline README.md]: The install.sh is located inside of the install directory. So previous command either needs to be `cd quipucords-installer/install` or this command needs to be `./install/install.sh`.

File: README.md [hunk 2]
```diff
@@ -31,20 +32,90 @@ cd quipucords-installer
...
 ```
 git clone git@github.com:quipucords/quipucords-installer.git
 ```
-## <a name="issues"></a> Issues
+# <a name="test"></a> Test
+There are various options for testing your changes to the installation scripts.  You can test scripts from this repository or an official build.  First you need to launch the VMs configured with the installation files and docker images you wish to test.
+## Configuring the Virtual Machines
+First bring up the virtual machines using one of the methods below.
+
+### Method1: Testing local changes using a docker image build from source.
```
Comment (myersCody) [inline README.md]: You are missing a space `Method1:` => `Method 1:`

File: README.md [hunk 3]
```diff
@@ -31,20 +32,90 @@ cd quipucords-installer
...
+```
+
+## Installing QPC Inside the Virtual Machine
+The above `test-all` command will performa  `vagrant ssh`.  If you have no configuration help, then you can simply run `install.sh`.
+
+### Configuration Scripts (internal)
+Create or obtain a tarball named `installer_config.tar.gz`.  The files in this tarball will automatically be copied inside the VMs mapped volumes.
+
+To configure use our internal scripts, do the following:
+1. cd /quipucords_installer;sudo su
```
Comment (myersCody) [inline README.md]: I think these steps should look like the rest of the code. Wrapped in a ``` cd /quipucords_installer; sudo su ```

File: README.md [hunk 4]
```diff
@@ -31,20 +32,90 @@ cd quipucords-installer
...
+```
+
+To test your local scripts on all supported OS's run the following.
+```
+make setup-release version=0.0.46
+make test-all
+```
+
+## Installing QPC Inside the Virtual Machine
+The above `test-all` command will performa  `vagrant ssh`.  If you have no configuration help, then you can simply run `install.sh`.
```
Comment (Dostonbek1) [inline README.md]: Typo: the 'performa' should change to 'perform'

File: README.md [hunk 5]
```diff
@@ -31,20 +32,90 @@ cd quipucords-installer
...
+### Method1: Testing local changes using a docker image build from source.
+First obtain all the required repositories.
+```
+git clone git@github.com:quipucords/quipucords.git
+git clone git@github.com:quipucords/quipucords-ui.git
+git clone git@github.com:quipucords/quipucords-installer.git
+```
+Build the docker image and test with the local install scripts on all supported OS's run the following.
+```
+make build-docker
```
Comment (myersCody) [inline README.md]: I get an error when I run this command:
```
cd ../quipucords;docker -D build . -t quipucords:
invalid argument "quipucords:" for t: invalid reference format
See 'docker build --help'.
make: *** [build-docker] Error 125
```

It seems like I am missing the `$(version)` variable in the makefile. Do I need to pass it in? 
```
cd ../quipucords;docker -D build . -t quipucords:$(version)
```

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,93 @@
+DATE = $(shell date)
+TOPDIR = $(shell pwd)
+
+help:
+	@echo "Please use \`make <target>' where <target> is one of:"
+	@echo "  setup                                          Copy configuration, install, packages to OS specific folders"
+	@echo "  setup-release version=<release_version>        Download official install scripts from GitHub specified by version.  Copy configuration, install, packages to OS specific folders"
+	@echo "  refresh                                        Recopy configuration, install, packages to OS specific folders"
+	@echo "  build-docker                                   Build a docker image for quipucords source"
```
Comment (myersCody) [inline Makefile]: We should update the help to include the version variable:
```
make build-docker version=<ver>
```

Comment (kholdaway) [reply Makefile]: Good catch

File: README.md (hunk 1 above)
Comment (kholdaway) [reply README.md]: Fixed that too

File: README.md (hunk 2 above)
Comment (kholdaway) [reply README.md]: Thanks

File: README.md (hunk 5 above)
Comment (kholdaway) [reply README.md]: The version is required (as you indicated in the make help above).  I added the version to the command.

File: README.md (hunk 4 above)
Comment (kholdaway) [reply README.md]: Thanks!

File: README.md (hunk 3 above)
Comment (kholdaway) [reply README.md]: Ok.

**Pull request merged**